docker-compose-dev.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
+
+services:
+    rif-relay:
+        build:
+            context: .
+        entrypoint: 'node dist/commands/Start.js --config server-config.testnet.json'
+        ports:
+            - '8090:8090'
+        volumes:
+            - './environment:/srv/app/environment'
+        networks:
+            - rif-relay
+
+networks:
+    rif-relay:
+        driver: bridge
```

Comment:
I'm not sure if we need this. Why did you add it?

Reply:
Actually not necessary, forgot to remove it during debugging.